lib/kiev/shoryuken/message_tracer.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Kiev
+  module Shoryuken
+    class MessageTracer
+      def call(options)
+        attrbutes = options[:message_attributes] ||= {}
+        SubrequestHelper.payload.each do |key, value|
+          attrbutes[key] = {
+            data_type: "String",
```

Comment:
this will encode `request_depth` as string, which means it will be presented as string in JSON, and lead to index failure in ElasticSearch.

Reply:
Not quite. `RequestId` middleware has to increment it or set to zero.
So the output is actually an integer.
I'll have a test that verifies this.